assets/telekom/fonts/telekom-variable-fonts.css

```diff
@@ -0,0 +1,9 @@
+@font-face {
+	font-family: "TeleNeoWeb";
+	font-style: normal;
+	font-weight: 200 800;
+	font-stretch: 75% 125%;
+	/* src: url("./TeleNeoWeb/TeleNeoVariable-Upright.woff2") format("woff2-variations"); */
+	src: url("./fonts/TeleNeoWeb/TeleNeoVariable-Upright.woff2") format("woff2-variations");
```

Comment:
@felix-ico the commented out one, in line 6, looks like the correct one to me…, no? 🤔